src/AzureDevOpsPolicyConfigurator/Helpers/PolicyExtension.cs

```diff
@@ -80,6 +80,38 @@ public static bool DoesSubTypeMatch(this PolicyConfiguration configuration, Poli
             return configuration.Settings.Value<string>(policy.SubTypePropertyName) == policy.SubTypePropertyValue;
         }
 
+        /// <summary>
+        /// Flattens the policies by branches.
+        /// </summary>
+        /// <param name="policies">Policies</param>
+        /// <param name="serializer">Json serializer</param>
+        /// <returns>Flattened policies</returns>
+        public static IEnumerable<Policy> FlattenBranches(this IEnumerable<Policy> policies, IJsonSerializer serializer)
+        {
+            var policiesToFlatten = policies.Where(x => x.Branches != null && x.Branches.Count > 0).ToList();
```

Comment:
Use any?

```suggestion
            var policiesToFlatten = policies.Where(x => x.Branches != null && x.Branches.Any()).ToList();
```

Reply:
It's a list, any would read into it count is already there, therefore count > 0 better imho.